src/components/FormModal.js

```diff
@@ -14,13 +14,24 @@ export default class FormModal extends Modal {
     this._inputList = this._modalElement.querySelectorAll(
       validationOptions.inputSelector
     );
+    this._submitButtonText = this._submitButton.textContent;
+  }
+
+  renderLoading(isLoading, loadingText = "Saving...") {
```

Comment:
Great that you've implemented the functionality for rendering loading via the class methods